src/Silk.Core/Commands/Tests/TagCommand.cs

```diff
@@ -0,0 +1,270 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using DSharpPlus.CommandsNext;
+using DSharpPlus.CommandsNext.Attributes;
+using DSharpPlus.Entities;
+using DSharpPlus.Interactivity.Enums;
+using DSharpPlus.Interactivity.Extensions;
+using MediatR;
+using Silk.Core.Services;
+using Silk.Core.Utilities.HelpFormatter;
+using Silk.Data.MediatR;
+using Silk.Data.Models;
+using Silk.Extensions;
+
+namespace Silk.Core.Commands.Tests
+{
+    [RequireGuild]
+    [Group("tag")]
+    [Category(Categories.Server)]
+    public class TagCommand : BaseCommandModule
+    {
+        private readonly IMediator _mediator;
+        private readonly TagService _tagService;
+        private readonly string[] _reservedWords = new[]
```

Comment:
Consider generating this instead of hardcoding if possible

Reply:
how so?

Reply:
Reflection, perhaps?

Reply:
Maybe, there's a way how D#+ gets command names so I'm sure you can fill that array in the ctor using whatever D$+ does.

Reply:
I suppose. Seems like it would be a bit clunky, but I'll think on that one.